src/web/src/components/AboutFooter.test.tsx

```diff
@@ -2,6 +2,11 @@ import { render, cleanup } from '@testing-library/react';
 
 afterEach(cleanup);
 
+// Make sure we get a consistent date regardless of when the test is run
+const mockDate = new Date('2022-01-03T18:30:11.041Z');
+jest.spyOn(global, 'Date').mockImplementation(() => mockDate as any);
+Date.now = jest.fn(() => mockDate.getTime());
```

Comment:
This is even better, eliminating the need for `milliseconds` altogether.  Nice improvement.